src/days-2-4-adv/player.py

```diff
@@ -2,6 +2,38 @@
 # currently.
 
 class Player:
-    def __init__(self, title, room):
+    def __init__(self, title, current_room):
         self.title = title
-        self.room = room
+        self.current_room = current_room
+
+    def move(self, getDir):
+        if getDir == 'q':
+            print('See you later!')
+        if self.current_room == 'outside':
+            if getDir == 'n':
+                self.current_room = 'foyer'
+            else:
+                print('Wrong direction, please re-input!')
+        elif self.current_room == 'foyer':
+            if getDir == 'n':
+                self.current_room = 'overlook'
+            elif getDir == 'e':
+                self.current_room = 'narrow'
+            else:
+                print('Wrong direction, please re-input!')
+        elif self.current_room == 'overlook' or self.current_room == 'treasure':
```

Comment:
you might need an elif if self.current_room is "narrow" because it looks like a user can go into narrow, but there is no logic for directions to go from there.

Reply:
Also, I don't think you can use a conditional "or", because only a player can only go south from overlook to arrive at "foyer". If player travels south from "treasure", they arrive at "narrow".